osgp/protocol-adapter-dlms/osgp-protocol-adapter-dlms/src/test/java/org/opensmartgridplatform/adapter/protocol/dlms/domain/commands/datetime/SynchronizeTimeCommandExecutorTest.java

```diff
@@ -91,6 +96,10 @@ void testSynchronizeTime() throws ProtocolAdapterException, IOException {
     assertThat(resultCode).isEqualTo(AccessResultCode.SUCCESS);
     verify(this.dlmsConnection, times(1)).set(this.setParameterArgumentCaptor.capture());
 
+    // save timezone
```

Comment:
Add test for null value timezone

Reply:
Timezone is required in SynchronizeTimeRequest. No need to test it here
`@XmlElement(name = "TimeZone", required = true)`